package Fundamentals.Exception_Handling;

public class Demo_Exception
{
    public static void main(String[] args)
    {
        int data=500;
        int value=0;
        int res=data/value;
        System.out.println(res);//It output as Arithmetic Exception by using try and catch method it can be execute the error
    }
}
